csv-to-genesis/src/test/kotlin/jp/co/soramitsu/Tests.kt

```diff
@@ -0,0 +1,66 @@
+package jp.co.soramitsu
+
+import jp.co.soramitsu.iroha2.Genesis
+import jp.co.soramitsu.iroha2.client.Iroha2Client
+import jp.co.soramitsu.iroha2.engine.IrohaTest
+import jp.co.soramitsu.iroha2.engine.WithIroha
+import jp.co.soramitsu.iroha2.generated.core.genesis.RawGenesisBlock
+import jp.co.soramitsu.iroha2.query.QueryBuilder
+import jp.co.soramitsu.iroha2.testcontainers.IrohaContainer
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Timeout
+import java.io.File
+
+@Timeout(300)
+class Tests : IrohaTest<Iroha2Client>() {
+
+    private val converter = Converter()
+
+    @Test
+    fun `should convert csv to genesis and run Iroha with it`(): Unit = runBlocking {
+        val genesis = converter.toGenesis(
+            File("src/main/resources/example.csv"),
+            File("src/main/resources/example_genesis.json")
+        )
+        startContainer(genesis).use { container ->
+            val client = Iroha2Client(container.getApiUrl())
+
+            QueryBuilder.findAllDomains()
+                .account(Converter.ADMIN_ID)
+                .buildSigned(Converter.ADMIN_KEYPAIR)
+                .let { client.sendQuery(it) }
+                .also { domains -> assert(domains.size == 3) }
```

Comment:
what's the logic behind, why just checking 3 domains, not amount of assets for example